app/main/routes.py

```diff
@@ -100,3 +113,17 @@ def http_exception(error):
 def csrf_error(error):
     flash("The form you were submitting has expired. Please try again.")
     return redirect(request.full_path)
+
+
+@bp.before_request
```

Comment:
How does this handle requests going to other blueprints?

Reply:
Decided to use nested blue prints, so this should not be a problem